src/service/proxy.js

```diff
@@ -110,4 +110,21 @@ export default class Proxy {
     this._requests = [];
   }
 
+  /**
+   * Adjusts the proxy to throttle the overall connection speed to the provided kilobytes/second.
+   * @param rate maximum overall speed in kilobytes/second
+   */
+  slow(rate) {
+    rate *= 1024;
+    this._proxy.slow({rate});
+  }
+
+  /**
+   * Disables throttling, allowing the proxy to run at maximum speed
+   */
+  relax() {
```

Comment:
A better name? Probably, I'm up for suggestions, but the only others that came to mind were:
* open
* dethrottle
* clear
* flood